lib/scores.js

```diff
@@ -0,0 +1,114 @@
+const Async = require('async');
+const level = require('../lib/level.js');
```

Comment:
@nelsonic  We definitely were thinking about using Redis eventually, Alan just suggested LevelDB as a simple way to get started without having to worry about deploying anything new. We've got an open issue reminding us to consider other options when scaling: https://github.com/TheScienceMuseum/collectionsonline/issues/339, I'll add the Redis suggestion